### ISSUE TITLE
Documentation!

### ISSUE THREAD

**Issue by nhcarrigan** [eddiejaoude/stargate#7]
We will need the following documentation:

- Contributing guidelines
- Privacy Policy (required for Discord bots)
- Flesh out readme
- Usage documentation

**Comment by Vyvy-vi** [comment]
Since, Privacy Policies are mentioned. Here are a few autogeneraters for it:

https://app-privacy-policy-generator.firebaseapp.com/

https://termly.io/products/privacy-policy-generator/

https://getterms.io/

https://www.shopify.com/tools/policy-generator


And a reference to Nick's Policy(it's good):
https://github.com/nhcarrigan/Becca-Lyria/blob/main/PRIVACY.md

My bot's privacy policy(inspired from Nick's policy):
https://github.com/Vyvy-vi/TearDrops/blob/docs(197)-privacy-policy/legal/PRIVACY.md

**Comment by moeinio** [comment]
some additional info is needed in order for this to be generated, things like: email address, phone number, company name, etc...

**Comment by Vyvy-vi** [comment]
Yes, I think this would be best made by @eddiejaoude or the clients.
esp. since this is considering legal issues and might need their insights for this.
